src/component/marker/MarkLineView.js

```diff
@@ -44,14 +44,14 @@ var markLineTransform = function (seriesModel, coordSys, mlModel, item) {
         var value;
 
         if (item.yAxis != null || item.xAxis != null) {
-            valueDataDim = item.yAxis != null ? 'y' : 'x';
+            valueDataDim = seriesModel.get('stack') != null ? '_\0ecstackresult' : item.yAxis != null ? 'y' : 'x';
```

Comment:
I think `'_\0ecstackresult'` should be set somewhere as a constant because it's also used in `dataStackHelper.js`.
How about exporting a constant called `stackResultDim`? @100pah

Reply:
Also, why using '__\0ecstackresult' not just '__ecstackresult'?